<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParadigmWindow</class>
 <widget class="QDialog" name="ParadigmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Информация о регулярной форме</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Формы слова</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QTableView" name="dictionaryTable"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Проиграть запись выбранного слова</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>92</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="title">
         <string>Добавить форму слова в словарь</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QFrame" name="frameWord">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>143</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelWordForm">
            <property name="text">
             <string>Слово</string>
            </property>
            <property name="buddy">
             <cstring>wordForm</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="wordForm"/>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="transcriptionFrame">
         <property name="geometry">
          <rect>
           <x>159</x>
           <y>23</y>
           <width>143</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labelTranscriptionForm">
            <property name="text">
             <string>Транскрипция</string>
            </property>
            <property name="buddy">
             <cstring>transcritptionForm</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="transcritptionForm"/>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frameTranslation">
         <property name="geometry">
          <rect>
           <x>308</x>
           <y>23</y>
           <width>143</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelTranslation">
            <property name="text">
             <string>Перевод</string>
            </property>
            <property name="buddy">
             <cstring>translationForm</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="translationForm"/>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frameSound">
         <property name="geometry">
          <rect>
           <x>457</x>
           <y>23</y>
           <width>106</width>
           <height>58</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="labelSound">
            <property name="text">
             <string>Звуковой файл</string>
            </property>
            <property name="buddy">
             <cstring>soundButton</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="soundButton">
            <property name="text">
             <string>Выбрать файл</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="framePraat">
         <property name="geometry">
          <rect>
           <x>569</x>
           <y>23</y>
           <width>114</width>
           <height>58</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="labelPraat">
            <property name="text">
             <string>Файлы для Praat</string>
            </property>
            <property name="buddy">
             <cstring>praatButton</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="praatButton">
            <property name="text">
             <string>Выбрать файлы</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frameSubmit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>689</x>
           <y>23</y>
           <width>262</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="submitButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>23</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Добавить форму слова в словарь</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
